src/60_Samples_%26_Templates/Housing.html

```diff
@@ -202,41 +233,41 @@ <h4>Floorplan</h4>
         </section>
         <section>
           <h4>Mortgage Calculator</h4>
```

Comment:
Idea: should we place the mortgage calculator outside the accordion to make it more prominent? We could give it it's own section with heading.

Reply:
done